src/network/networkd-manager.c

```diff
@@ -313,6 +313,9 @@ static int manager_rtnl_process_link(sd_netlink *rtnl, sd_netlink_message *messa
                         /* link is new, so add it */
                         r = link_add(m, message, &link);
                         if (r < 0) {
+                                if (r == -ENODEV)
+                                        link_drop(link);
```

Comment:
Mmh this change does not seem related to the rest? Or is it?

Reply:
Ouch I pushed in the wrong tab.